core/src/v2/models.rs

```diff
@@ -107,6 +107,18 @@ pub struct Api<P, R, S> {
     pub parameters: BTreeMap<String, P>,
     #[serde(default = "BTreeMap::new", skip_serializing_if = "BTreeMap::is_empty")]
     pub responses: BTreeMap<String, R>,
+    #[serde(
+        default,
+        rename = "securityDefinitions",
+        skip_serializing_if = "BTreeMap::is_empty"
+    )]
+    pub security_definitions: BTreeMap<String, SecurityScheme>,
+    #[serde(default, skip_serializing_if = "BTreeMap::is_empty")]
+    pub security: BTreeMap<String, Vec<String>>,
+    #[serde(skip_serializing_if = "Vec::is_empty")]
```

Comment:
Please add `default` to serde attributes. That's what's causing the build failure.